<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDialog</class>
 <widget class="QDialog" name="UserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>884</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1121</width>
     <height>841</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Welcome</string>
    </attribute>
    <widget class="QLabel" name="image_admin">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>1111</width>
       <height>781</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Profil</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>491</width>
       <height>801</height>
      </rect>
     </property>
     <property name="title">
      <string>User informations :</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="label_64">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Role :</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_56">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>461</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_67">
       <item>
        <widget class="QLabel" name="label_66">
         <property name="text">
          <string>ID : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ID_2">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_65">
         <property name="text">
          <string>Username :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_username_3">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_61">
         <property name="text">
          <string>Email Adress :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_email_3">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_52">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>80</y>
        <width>461</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_63">
       <item>
        <widget class="QLabel" name="label_55">
         <property name="text">
          <string>Surnoun: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_surnoun_3">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_56">
         <property name="text">
          <string>Noun :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_noun_3">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_60">
         <property name="text">
          <string>CIN : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_cin_3">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_57">
         <property name="text">
          <string>Birth date :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="date_3">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_74">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>530</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>User History : </string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_his_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>550</y>
        <width>481</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_75">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Current loaned books : </string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_his_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>481</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_76">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>439</y>
        <width>27</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Ref : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ref_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>440</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_23">
      <property name="geometry">
       <rect>
        <x>264</x>
        <y>440</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Return book</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>266</width>
        <height>67</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_58">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="radioButton_admin_3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Admin</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_teacher_3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Teacher</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_student_3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Student</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_61">
           <item>
            <widget class="QLabel" name="label_58">
             <property name="text">
              <string>Speciality :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_spe_3">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_62">
           <item>
            <widget class="QLabel" name="label_59">
             <property name="text">
              <string>Acadamec Year :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_EDIT_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="editable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>340</y>
       <width>611</width>
       <height>441</height>
      </rect>
     </property>
     <property name="title">
      <string>Doc's info</string>
     </property>
     <widget class="QWidget" name="layoutWidget_49">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>591</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_65">
       <item>
        <widget class="QLabel" name="label_62">
         <property name="text">
          <string>Ref : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ref_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_63">
         <property name="text">
          <string>Title : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_title_4">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_66">
         <item>
          <widget class="QLabel" name="label_67">
           <property name="text">
            <string>Author :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_author_4">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_68">
           <property name="text">
            <string>Category :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_cathegory_4">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit_des_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>231</width>
        <height>111</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_69">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Description : </string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_his_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>591</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_70">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Loan History : </string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_53">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>591</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_68">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_69">
         <item>
          <widget class="QLabel" name="label_71">
           <property name="text">
            <string>Number of copies  :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_nbr_copie_4">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_70">
         <item>
          <widget class="QLabel" name="label_72">
           <property name="text">
            <string>Publish Year :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_year_4">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_71">
         <item>
          <widget class="QLabel" name="label_73">
           <property name="text">
            <string>Number of pages : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_nbr_page_4">
           <property name="echoMode">
            <enum>QLineEdit::Normal</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_54">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>140</y>
        <width>316</width>
        <height>67</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_72">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QRadioButton" name="radioButton_acproj_4">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Academic Project</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_book_4">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Book</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_journal_4">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Journal</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_73">
           <item>
            <widget class="QLabel" name="label_hp_sup_4">
             <property name="text">
              <string>Supervisor: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_hp_sup_4">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_74">
           <item>
            <widget class="QLabel" name="label_sub_4">
             <property name="text">
              <string>Subject : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_sub_4">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_22">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>400</y>
        <width>231</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Loan Book</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>0</y>
       <width>611</width>
       <height>341</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QWidget" name="layoutWidget_34">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>291</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_59">
       <item>
        <widget class="QComboBox" name="comboBox_search_doc_2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="search_3"/>
       </item>
       <item>
        <widget class="QPushButton" name="Search_doc_2">
         <property name="text">
          <string>SEARCH</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableView" name="tableView_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>591</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_48">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>551</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_60">
       <item>
        <widget class="QPushButton" name="pushButton_15">
         <property name="text">
          <string>Load Books' Table</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_16">
         <property name="text">
          <string>Load Journals' Table</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_17">
         <property name="text">
          <string>Load Academic projects' Table</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_18">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>290</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Table</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>850</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
